yt_dlp/extractor/bbc.py

```diff
@@ -1255,6 +1268,48 @@ def extract_all(pattern):
                 lambda s: self._parse_json(s, playlist_id, fatal=False),
                 re.findall(pattern, webpage))))
 
+        def parse_model(model):
+            '''Extract single video from model structure'''
+            if isinstance(model, list):
+                model = model[0]
+            item_id = traverse_obj(model, ('versions', 0, 'versionId', {str}))
+            if not item_id:
+                return
+            formats, subtitles = self._download_media_selector(item_id)
+            return {
+                'id': item_id,
+                'formats': formats,
+                'subtitles': subtitles,
+                **traverse_obj(model, {
+                    'title': ('title', {str}),
+                    'thumbnail': ('imageUrl', {lambda u: urljoin(url, u.replace('$recipe', 'raw'))}),
+                    'description': (
+                        'synopses', ('long', 'medium', 'short'), {str}, any),
+                    'duration': ('versions', 0, 'duration', {int}),
+                    'timestamp': ('versions', 0, 'availableFrom', {lambda x: int_or_none(x, scale=1000)}),
+                })
+            }
+
+        # US accessed article with single embedded video (e.g.
+        # https://www.bbc.com/news/uk-68546268)
+        next_data = traverse_obj(self._search_nextjs_data(webpage, playlist_id, default={}), (
+            'props', 'pageProps', 'page'))
+        model = traverse_obj(next_data, (
+            ..., 'contents', lambda _, v: v['type'] == 'video',
+            'model', 'blocks', lambda _, v: v['type'] == 'media',
+            'model', 'blocks', lambda _, v: v['type'] == 'mediaMetadata',
+            'model'))
```

Comment:
Extract 0th item from list:
```suggestion
        model = traverse_obj(next_data, (
            ..., 'contents', lambda _, v: v['type'] == 'video',
            'model', 'blocks', lambda _, v: v['type'] == 'media',
            'model', 'blocks', lambda _, v: v['type'] == 'mediaMetadata',
            'model', (0, None), {dict}, any)
```